src/ci/bin/test.inc.sh

```diff
@@ -1278,8 +1362,10 @@ cromwell::build::setup_common_environment() {
             cromwell::private::delete_boto_config
             cromwell::private::delete_sbt_boot
             cromwell::private::upgrade_pip
-            cromwell::private::pull_common_docker_images
-            cromwell::private::start_docker_databases
+            if [[ "${BUILD_TYPE}" != "sbt" ]]; then
+              cromwell::private::pull_common_docker_images
+              cromwell::private::start_docker_databases
```

Comment:
TOL probably some other `BUILD_TYPE`s like metadata comparison and manifest creator also don't care about these

Reply:
Actually I think there's a chance this if statement is not required at all for CircleCI

Reply:
I feel like the cost of starting the DBs is low and we can reduce complexity by having things be less specific. Just a loosely held thought.